pyproject.toml

```diff
@@ -9,7 +9,7 @@ authors = [
     {name = "Google LLC", email = "googleapis-packages@google.com"}
 ]
 dependencies = [
-    "google-cloud-alloydb-connector[asyncpg]>=1.0.0, <2.0.0",
+    "google-cloud-alloydb-connector[asyncpg]>=0.4.0, <2.0.0",
```

Comment:
is this a mistake?

Reply:
@jackwotherspoon This was accidentally updated. The library should be able to support from v0.4 to present.

Reply:
the native support for the IP type as a `str` was only added in `1.0.0` of the AlloyDB Python Connector. Since we removed the conversion from this lib as part of https://github.com/googleapis/langchain-google-alloydb-pg-python/pull/102, Python Connector version 1.0.0 or greater is required.

Reply:
Now updated.